### DISCUSSION TITLE
I cant get Firefly to work

### DISCUSSION THREAD

**Discussion by riibas4** [firefly-iii/firefly-iii#8172] [Firefly III]
Whenever i open Firefly-iii on the browser, it takes to a page named: xxx.xxx.x.x/public/login (the ip is blurred out)
Not Found
The requested URL was not found on this server. That's what I get and I dont know how to fix. So PLEASE, help me... I've been trying for 3 days and I just can't..

**Comment by JC5** [comment]
Could you not use Docker instead?

**Reply by riibas4** [reply]
I'm trying to run it using a Remote access proxmox machine. I'm not really sure how I can set docker up (if it is even possible).  All I know is that it was already painful enough to get where I am 💀
I just used the Self Hosted steps on the documentation and it didn't work that well

**Reply by JC5** [reply]
Yes I see that but Docker may be a lot more straight-forward, you've shared zero details about your setup so I'm not sure how to debug this.

**Reply by riibas4** [reply]
I mean, what do you want to know? I got a debian 12 machine running remotely trough proxmox. I installed sudo and all the stuff it said on the Documentation and now I'm getting this error... Anything u need to know just ask away

**Reply by JC5** [reply]
> Anything u need to know just ask away

This is not how it works. I'm not here to play "20 questions" with you. This is my Friday night too.

**Reply by riibas4** [reply]
well, all that I know is what I said already.. debian 12, no UI version (so only commands) running on proxmox. Brand New machine. I installed sudo commands. Then I followed every step on the self Hosted documentation. That's all I did
Then when I access 192.168.0.8/ it says WARMING firefly this shouldn't be public whatever. Then it redirects me to 192.168.0.8/public/login, which doesn't exist. I'm not trying to come off as rude, so I'm sorry if it sounds like it

**Reply by riibas4** [reply]
@JC5 Still think that docker is better? My setup is: Proxmox virtual environment running Debian 12, but I can only use commands (there is no UI). As of now, I don't plan on actually publishing the website, only using it myself. I followed all of the steps and it all worked great, but now when I try to access it the login page doesn't exist. The DB is set up correctly as far as I know. The user and password in the .env file is correct. Oh and btw, sorry for the way i spoke in friday, i was a bit angry...

**Comment by BackSeat** [comment]
The problem isn't Firefly. The problem is that you don't know enough about Linux, networking, Apache, etc, to manage a self-hosted system. Once you get it working, how will you set up and manage HTTPS? How will you backup the database? What will you do to keep it secure? You don't help yourself when you can't be bothered to copy messages correctly - it doesn't say "WARMING" nor does it say "this shouldn't be public whatever".

Here's a clue to get you started: Google "RFC1918". The Wikipedia page is good enough. Read it, understand it, and you'll be a lot closer to having this working. My advice, though, would be to team up with someone who understands this (try Googling for your local Linux User Group where you'll find people who will help you in exchange for a beer or two).

**Reply by riibas4** [reply]
I didn't copy it properly cuz I'm not at work anymore, and it was a simple typo.  I just can't remember letter by letter what it said ahaha. Anyways, what does that have to do with the Login folder not existing?
